src/openexr.imageio/exrinput.cpp

```diff
@@ -319,7 +320,9 @@ void set_exr_threads ()
 
     int oiio_threads = 1;
     OIIO::getattribute ("exr_threads", oiio_threads);
-
+    if (oiio_threads == 0) {
+        oiio_threads = Sysutil::hardware_concurrency();
```

Comment:
The default value (if they never set the attribute) is physical_concurrency(), which is the number of true cores. The hardware_concurrency, on some processors may be a higher number of virtual (hyperthreaded) cores.

It's hard to know which is "best" as the default. But I think they should be the same whether it's the default or explicitly set to 0.

Do you have strong feelings about hardware_concurrency versus physical_concurrency? My rationale was that using h_c rarely had a big gain in performance, but took the chance of "over-threading", so I erred on the side of conservatively picking p_c. But I could be talked into a different position.